lldb/source/Target/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -415,7 +415,7 @@ class LLDBMemoryReader : public swift::remote::MemoryReader {
     Target &target(m_process.GetTarget());
     Address addr(address.getAddressData());
     Status error;
-    if (size > target.ReadMemory(addr, dest, size, error, true)) {
+    if (size > target.ReadMemory(addr, dest, size, error)) {
```

Comment:
```
bool bool force_live_memory = false;
if (size > target.ReadMemory(addr, dest, size, error, force_live_memory)) 
```
is perhaps more readable?

Would there be any reason to make this controllable via a setting?

Reply:
Making it controllable via a setting was my initial implementation on https://reviews.llvm.org/D100338. @jasonmolenda argued that reading from the file-cache when read-only should be the default, I think this makes sense since, if we're doing things correctly, all these file-cache reads should be safe. What do you think?

Reply:
And because the flag only affects read-only sections, it wouldn't have any effect on queries where we read, e.g., an object pointer from the __data section or the heap. O think that makes sense. Thanks for clarifying!